packages/dcos-integration-test/extra/test_dcos_checks.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import uuid
+
+
+def test_dcos_checks_detect_ip(dcos_api_session):
+    cmd = [
```

Comment:
A description (docstring) will be helpful. 
```Exercise dcos-check so that pointing to a invalid detect-ip script like --detect-ip /etc/hosts below raises an Exception.```